job-service-config/src/main/java/com/github/jobservice/configuration/AppConfig.java

```diff
@@ -38,15 +39,23 @@
 @Configuration
 @PropertySource(value = "classpath:${JOB_SERVICE_API_CONFIG_PATH:config.properties}", ignoreResourceNotFound = true)
 public class AppConfig {
+    private static final String JOB_SERVICE_SECRETS_CACHE_COUNT_CONFIG = "JOB_SERVICE_SECRETS_CACHE_COUNT";
+    private static final int JOB_SERVICE_SECRETS_CACHE_COUNT_DEFAULT = 20;
 
     private static final LoadingCache<String, String> SECRETS_CACHE = CacheBuilder.newBuilder()
+            .maximumSize(getEnvOrDefault(JOB_SERVICE_SECRETS_CACHE_COUNT_CONFIG, JOB_SERVICE_SECRETS_CACHE_COUNT_DEFAULT))
```

Comment:
I'll approve but tbh I don't think this needs to be configurable at all.  Looking at the code there are just two possible keys at the minute.  It does no harm to use a slightly higher value in case more secrets are added but I just can't think when or why anyone would ever adjust this configuration entry.